react/src/js/react/posts/Post.jsx

```diff
@@ -39,48 +42,58 @@ export default class Post extends Reflux.Component {
     }
 
     renderFooter() {
-        const { post } = this.props;
+        const commentsLength = this.currentPost.comments.length;
         return (
             <div className="post-footer">
-                <p className="categories">{post.categories}</p>
-                { post.source ? <a href={post.source} className="source">source</a> : null}
-                { post.origin ? <a href={post.origin}>origin</a> : null }
+                <p className="categories">{this.currentPost.categories}</p>
+                { this.currentPost.source ? <a href={this.currentPost.source} className="source">source</a> : null}
```

Comment:
I don't think we need to display the source and origin URLs, it doesn't seem important from a user's perspective